crates/release-automation/src/lib/tests/cli.rs

```diff
@@ -957,7 +957,7 @@ fn release_dry_run_fails_on_unallowed_conditions() {
             "--log-level=debug",
             "release",
             "--dry-run",
-            "--allowed-semver-increment-modes=!pre_patch beta-dev",
+            "--allowed-semver-increment-modes=!pre_patch dev",
```

Comment:
i'm confused about this not throwing an error. the test is supposed to act on a mocked workspace and shouldn't need adjustment :thinking:

Reply:
looking further down, this test asserts that there's a failure which is still given with this change. it is an unnecessary change though. should we rename it something obviously wrong so that it doesn't cause confusion in the future?

```suggestion
            "--allowed-semver-increment-modes=!pre_patch definitenotallowed",
```

Reply:
actually i prefer just leaving it alone for now as we're acting on a release branch

```suggestion
            "--allowed-semver-increment-modes=!pre_patch beta-dev",
```